recipes/starshot_blog/config/node.type.blog.yml

```diff
@@ -17,6 +17,13 @@ third_party_settings:
     unpublish_enable: true
     unpublish_required: false
     unpublish_revision: true
+  type_tray:
+    type_category: _none
+    type_thumbnail: ''
+    type_icon: /sites/default/files/type-tray-icons/icon-blog-post.svg
```

Comment:
🤔 My only concern here is that, if and when we add the ability to install the site in a directory other than `sites/default`, this will break.

So we either need test coverage that the icons actually appear, or we need to find a way to make this a little more dynamic. A custom config action, maybe? IMHO, the correct resolution here is for Type Tray to support more dynamism in this setting, if there's not already an issue for that.

That said, I'm okay deferring this to a follow-up PR. (Can you open an issue for it here?)